NachoSpigot-Server/src/main/java/net/minecraft/server/MinecraftServer.java

```diff
@@ -555,6 +557,9 @@ public double getAverage() {
     }
     // PaperSpigot End
  
+    // Thread affinity - lock
+    AffinityLock lock = null;
+    
     public void run() {
```

Comment:
```suggestion
    
    public void run() {
        AffinityLock lock = null; // Nacho
```

Reply:
This should be keep out of the method since MinecraftServer itself is a runnable so you're creating a AffinityLock instance for every run by doing this